app/talk/comment.cjsx

```diff
@@ -76,7 +78,7 @@ module?.exports = React.createClass
     <div className="talk-comment">
       <div className="talk-comment-author">
         <PromiseRenderer pending={@avatarPending} promise={apiClient.type('users').get(id: @props.data.user_id).index(0)}>{(commentOwner) =>
-          <img src={commentOwner.avatar} alt={commentOwner.display_name}/>
+          <img src={commentOwner.avatar ? DEFAULT_AVATAR} alt={commentOwner.display_name}/>
```

Comment:
Eh, sorry, avatars are actually a separate link now: `user.get('avatar').then((avatar) -> avatar.src).catch(-> DEFAULT_AVATAR)`

It should also be cached manually at the component level since requesting the avatar of a user without one throws an error, which doesn't automatically cache. Avatars might be a nice thing to abstract into something like `<Avatar user={user} size="2em" />`.